models/cryptopunks/ethereum/cryptopunks_ethereum_events.sql

```diff
@@ -153,8 +150,8 @@ inner join {{ source('ethereum','transactions') }} tx on a.evt_tx_hash = tx.hash
 {% endif %}
 
 left join {{ source('prices', 'usd') }} p on p.minute = date_trunc('minute', a.evt_block_time)
-    AND p.contract_address = "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
-    AND p.blockchain = "ethereum"
+    and p.contract_address = "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
```

Comment:
in other scripts, you changed these to single quotes -- can you look to make consistent?